blog/rslint-a-extremely-fast-javascript-linter.md

```diff
@@ -0,0 +1,77 @@
+# RSLint - A extremely fast JavaScript linter
+
+ESLint is a de facto solution for [linting](https://en.wikipedia.org/wiki/Lint_%28software%29) JavaScript code. Also, `C` is the first that comes to our minds when we talk about compile some code from every kind. I know that a lot of compilers are compiled using the language that them compiles 😕. But when, we talk about parsers, abstract syntax trees (AST), and compile to binary: `C` comes to our minds. Maybe I'm old, I don't know. 🤔
```

Comment:
ESLint is a de facto solution for [linting](https://en.wikipedia.org/wiki/Lint_%28software%29) JavaScript code. Also, `C` is the first that comes to our minds when we talk about compile some code of every kind. I know that a lot of compilers are compiled using the language that they compile 😕. But when we talk about parsers, abstract syntax trees (AST), and compile to binary: `C` comes to our minds. Maybe I'm old, I don't know. 🤔